cluster-autoscaler/processors/processors.go

```diff
@@ -61,6 +62,8 @@ type AutoscalingProcessors struct {
 	CustomResourcesProcessor customresources.CustomResourcesProcessor
 	// ActionableClusterProcessor is interface defining whether the cluster is in an actionable state
 	ActionableClusterProcessor actionablecluster.ActionableClusterProcessor
+	// ScaleDownCandidatesObservers  is used to Update and Register new scale down candidates observer.
+	ScaleDownCandidatesObservers scaledowncandidates.CandidatesObserverProcessor
```

Comment:
nit: Wouldn't `ScaleDownCandidatesNotifier` better reflect the purpose of this object? (Probably worth aliging the type name then too.)